src/YieldDisburser.sol

```diff
@@ -155,9 +158,13 @@ contract YieldDisburser is OwnableUpgradeable {
                 uint256 vote = voterPower * voterDistribution[j];
                 projectDistributions[j] += vote;
                 totalVotes += vote;
+                projectVotes[j] += 1;
```

Comment:
Im confused why we'd use this value instead of the `projectDistributions` value for emitting `TotalVotesForProject` event. We want to know how many voters per project and not the vote distribution?

I also think it would be a good idea to just emit the event here when `i == breadchainVoters.length` instead of doing another loop over `projectCount` later on.

Reply:
Hmm, thoughts @subject026 ? Do we prefer to show the amount of voters or the total voting power? 

@bagelface I think there is some benefit to show the amount of humans as opposed the the abstract concept of voting power as it's derived here ...

Reply:
I guess. Technically this is just the number of addresses voting, which isn't necessary limited to one per human so that kind of makes it less meaningful.

Reply:
Also i'm wondering whether there is any reason we shouldn't emit just a single event for disbursement passing a 2D array or something. For example if we just did the wallet addresses and amounts like I described above:
```
[
  [0x12345678910, $250],
  [0x10987654321, £500],
  ...
]
```

Reply:
ok, amending this pr to include your suggestions!

Reply:
@subject026 Not condensing the event because then we lose the benefit of being able to give the entire history of distribution for a project on the project page

Reply:
conversation should be resolved with [ea93ba3](https://github.com/BreadchainCoop/breadchain/pull/27/commits/ea93ba30664d2ab622cf45945893cb67a02cd76b)